pyfar/classes/audio.py

```diff
@@ -108,6 +108,28 @@ def reshape(self, newshape):
 
         return reshaped
 
+    def transpose(self, *axes):
```

Comment:
I would suggest `axes=None` as default as in numpy

Reply:
I did that but then realized that this does not reproduce the behaviour of numpy. `np.transpose` uses `axes=None` per default, however, `np.ndarray.transpose` uses `*axes` which allows passing of tuples and multiple ints.

I have reverted these changes, left `axes` empty per default and also accept `None`.